src/main/default/lwc/voteTracker/voteTracker.css

```diff
@@ -0,0 +1,10 @@
+.title_bold {
+  font-weight: bold;
+}
+.text_light {
+  font-weight: 300;
+}
+.row {
```

Comment:
there's a css class from slds to align center stuff:
https://www.lightningdesignsystem.com/utilities/alignment/